monai/apps/deepgrow/interaction.py

```diff
@@ -51,8 +51,8 @@ def __init__(
         self.key_probability = key_probability
 
     def attach(self, engine: Union[SupervisedTrainer, SupervisedEvaluator]) -> None:
-        if not engine.has_event_handler(self, Events.ITERATION_STARTED):
-            engine.add_event_handler(Events.ITERATION_STARTED, self)
+        if not engine.has_event_handler(self, IterationEvents.ITERATION_STARTED):
```

Comment:
I think `IterationEvents` doesn't have `ITERATION_STARTED` event, no need to change this line.
Thanks.